source/nodejs/adaptivecards-designer/src/open-image-dialog.ts

```diff
@@ -273,24 +273,33 @@ export class OpenImageDialog extends Dialog {
         sampleImageTemplate.appendChild(message);
 
         this.fetchManager.getSampleImages().then((res) => {
-            const imageTitle = document.createElement("div");
-            imageTitle.innerText = "Choose from our Sample Images";
-            imageTitle.style.marginLeft = "15px";
-            imageTitle.className = "acd-image-title";
-            sampleImageTemplate.style.flexDirection = "row";
-            sampleImageTemplate.removeChild(spinnerElement);
-            sampleImageTemplate.removeChild(message);
-            sampleImageTemplate.appendChild(imageTitle);
-            const imageContainer = document.createElement("div");
-            imageContainer.className = "acd-sample-list";
-            for (let template of res.templates) {
-                let sampleImage = new ImageItem(template);
-                sampleImage.onClick = (selectedImage: string) => {
-                    this.renderImage(selectedImage);
-                };
-                imageContainer.appendChild(sampleImage.render());
+            if (res) {
+                const imageTitle = document.createElement("div");
+                imageTitle.innerText = "Choose from our Sample Images";
+                imageTitle.style.marginLeft = "15px";
+                imageTitle.className = "acd-image-title";
+                sampleImageTemplate.style.flexDirection = "row";
+                sampleImageTemplate.removeChild(spinnerElement);
+                sampleImageTemplate.removeChild(message);
+                sampleImageTemplate.appendChild(imageTitle);
+                const imageContainer = document.createElement("div");
+                imageContainer.className = "acd-sample-list";
+                for (let template of res.templates) {
+                    let sampleImage = new ImageItem(template);
+                    sampleImage.onClick = (selectedImage: string) => {
+                        this.renderImage(selectedImage);
+                    };
+                    imageContainer.appendChild(sampleImage.render());
+                }
+                sampleImageTemplate.appendChild(imageContainer);
+            } else {
+                const imageTitle = document.createElement("div");
```

Comment:
Shall we rename the constant as `errorElement` or `errorTitle` ?

Reply:
Updated the variable name. Addressed the review comments.